requirements-test.txt

```
@@ -0,0 +1,2 @@
+pytest==5.4.1
```

Include the project's requirements

```suggestion
-r ./requirements.txt

pytest==5.4.1
```